lib/node_modules/@stdlib/math/base/special/ceiln/src/ceiln.c

```diff
@@ -0,0 +1,95 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/math/base/special/ceiln.h"
+#include "stdlib/math/base/special/abs.h"
+#include "stdlib/constants/float64/max_base10_exponent.h"
+#include "stdlib/constants/float64/min_base10_exponent.h"
+#include "stdlib/constants/float64/max.h"
+#include <stdint.h>
+#include <math.h>
+
+/**
+* Rounds a numeric value to the nearest multiple of \\(10^n\\) toward positive infinity.
+*
+* @param x       input value
+* @param n       integer power of 10
+* @return        rounded value
+*
+* @example
+* // Round a value to 2 decimal places:
+* double y = stdlib_base_ceiln( 3.141592653589793, -2 );
+* // returns 3.15
+*
+* @example
+* // If n = 0, `ceiln` behaves like `ceil`:
+* double y = stdlib_base_ceiln( 3.141592653589793, 0 );
+* // returns 4.0
+*
+* @example
+* // Round a value to the nearest thousand:
+* double y = stdlib_base_ceiln( 12368.0, 3 );
+* // returns 13000.0
+*/
+double stdlib_base_ceiln( const double x, const int32_t n ) {
+    double s;
```

Comment:
Need to convert spaces to TABS.

Reply:
@steff456 Looks like this file stills needs to be converted to use TAB indentation.